docs/sources/operations/storage/boltdb-shipper.md

```diff
@@ -99,4 +106,25 @@ This problem would be faced even during rollouts which is quite common.
 To avoid this, Loki disables deduplication of index when the replication factor is greater than 1 and `boltdb-shipper` is an active or upcoming index type.
 While using `boltdb-shipper` please avoid configuring WriteDedupe cache since it is used purely for the index deduplication, so it would not be used anyways.
 
+### Compactor
+
+Compactor is a BoltDB Shipper specific service that reduces the index size by deduping the index and merging all the files to a single file per table.
+We recommend running a Compactor since a single Ingester creates 96 files per day which include a lot of duplicate index entries and querying multiple files per table adds up the overall query latency.
+
+*NOTE: There should be only 1 compactor instance running at a time that otherwise could create problems and may lead to data loss. *
```

Comment:
```suggestion
**Note:** There should be only 1 compactor instance running at a time that otherwise could create problems and may lead to data loss.
```